core/db_classes/EE_Line_Item.class.php

```diff
@@ -1414,12 +1704,13 @@ public function timestamp($raw = false)
      * @deprecated 4.6.0
      * @param string $type one of the constants on EEM_Line_Item
      * @return EE_Line_Item[]
+     * @throws EE_Error
      */
     protected function _get_descendants_of_type($type)
     {
         EE_Error::doing_it_wrong(
             'EE_Line_Item::_get_descendants_of_type()',
-            __('Method replaced with EEH_Line_Item::get_descendants_of_type()', 'event_espresso'),
+            esc_html__('Method replaced with EEH_Line_Item::get_descendants_of_type()', 'event_espresso'),
```

Comment:
The class and method name probably should not be localized.

Reply:
missed that, although doing so will break any existing translations, which I doubt exist for this text, so will change